conf/upf.json

```diff
@@ -146,6 +146,9 @@
 
     "": "p4rtc interface settings",
     "p4rtciface": {
+        "": "Set the UP4 slice identifier that this PFCP Agent instance belongs to. Default: 0",
+        "slice_id": 0,
```

Comment:
It defaults to zero because `Unmarshal` init all fields to zero, right?

Reply:
Correct. I'll double-check with Daniele if 0 is a correct default value for `slice_id`.

Reply:
ok, do the usual manual init in `config.go` in that case (non-zero) then.